app/src/main/java/org/systers/mentorship/models/Task.kt

```diff
@@ -18,7 +18,6 @@ import kotlinx.android.parcel.Parcelize
 data class Task(
         val id: Int,
         val description: String,
-        val isDone: Boolean,
```

Comment:
actually on viewing it again the code doesn't seem as re-useable. You don't have to remove the isDone from this model, just return the logic from here instead of the viewmodel.

Reply:
hmm, @vj-codes what do you think ?

Reply:
> actually on viewing it again the code doesn't seem as re-useable. You don't have to remove the isDone from this model, just return the logic from here instead of the viewmodel.

btw on what basis are you defining re-usability as the isDone field is getting removed from the api as it's redundant 🤔?

Reply:
> > actually on viewing it again the code doesn't seem as re-useable. You don't have to remove the isDone from this model, just return the logic from here instead of the viewmodel.
> 
> btw on what basis are you defining re-usability as the isDone field is getting removed from the api as it's redundant 🤔?

Reusability as in if we want to check if the task isDone anywhere else. You can try something like `val isDone = completedAt !=null`

Reply:
> > > actually on viewing it again the code doesn't seem as re-useable. You don't have to remove the isDone from this model, just return the logic from here instead of the viewmodel.
> > 
> > 
> > btw on what basis are you defining re-usability as the isDone field is getting removed from the api as it's redundant 🤔?
> 
> Reusability as in if we want to check if the task isDone anywhere else. You can try something like `val isDone = completedAt !=null`

maybe there also we can just check completedAt field, but let's see what @vj-codes thinks about this